.circleci/config.yml

```diff
@@ -231,11 +203,6 @@ jobs:
     working_directory: ~/api-platform/core
     steps:
       - checkout
-      - restore-mongodb-php-extension-build-cache:
-          php_version: << parameters.php_version >>
-      - install-mongodb-php-extension
```

Comment:
Rationale: Let PHPStan use our stubs so that we know when it's broken (time to sync changes from https://github.com/JetBrains/phpstorm-stubs/blob/master/mongodb/mongodb.php)... :pray: